lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -17,7 +17,7 @@
 DOCUMENTATION = r'''
 ---
 module: postgresql_idx
-short_description: Creates or drops indexes from a PostgreSQL database.
```

Comment:
See short descriptions of other modules. Most of them ends by ".", e.g. shell_module, postgresql_db, postgresql_user, etc.
Ok, it doesn't matter but I prefer sentences that end with "." )